lib/truncate.js

```diff
@@ -272,7 +287,10 @@ function truncMetricSet (metricset, opts) {
         ? opts.truncateKeywordsAt
         : opts.truncateStringsAt
 
-      return truncate(value, max)
+      if (value.length > max) {
+        return value.slice(0, max)
```

Comment:
Per discussion elsewhere, `slice` might have trouble with UTF-8 characters whose encoding requires more than three bytes. (🎉, etc.)

Reply:
Yup, we are breaking surrogate pairs. Boo.
A truncation function (`unitrunc()`) was added in commit f3c179fe7a337a9616f54a33632c7d1c64a15dc8 that is still fast and doesn't break surrogate pairs.

Here is the test failing **without** unitrunc:

```
...
# do not break surrogate pairs in truncation
ok 491 context.db.statement was truncated
not ok 492 context.db.statement was truncated without breaking a surrogate pair
  ---
    operator: equal
    expected: 'foo'
    actual:   'foo�'
    at: Test.<anonymous> (/Users/trentm/el/apm-nodejs-http-client2/test/truncate.js:470:5)
    stack: |-
      Error: context.db.statement was truncated without breaking a surrogate pair
          at Test.assert [as _assert] (/Users/trentm/el/apm-nodejs-http-client2/node_modules/tape/lib/test.js:228:54)
          at Test.bound [as _assert] (/Users/trentm/el/apm-nodejs-http-client2/node_modules/tape/lib/test.js:80:32)
          at Test.equal (/Users/trentm/el/apm-nodejs-http-client2/node_modules/tape/lib/test.js:389:10)
          at Test.bound [as equal] (/Users/trentm/el/apm-nodejs-http-client2/node_modules/tape/lib/test.js:80:32)
          at Test.<anonymous> (/Users/trentm/el/apm-nodejs-http-client2/test/truncate.js:470:5)
          at Test.bound [as _cb] (/Users/trentm/el/apm-nodejs-http-client2/node_modules/tape/lib/test.js:80:32)
          at Test.run (/Users/trentm/el/apm-nodejs-http-client2/node_modules/tape/lib/test.js:96:10)
          at Test.bound [as run] (/Users/trentm/el/apm-nodejs-http-client2/node_modules/tape/lib/test.js:80:32)
          at Immediate.next [as _onImmediate] (/Users/trentm/el/apm-nodejs-http-client2/node_modules/tape/lib/results.js:83:19)
          at processImmediate (internal/timers.js:461:21)
  ...

1..492
# tests 492
# pass  491
# fail  1
```